packages/live-share-inking/src/core/Colors.ts

```diff
@@ -0,0 +1,154 @@
+    if (color) {
+        const regEx = /#([0-9A-F]{2})([0-9A-F]{2})([0-9A-F]{2})/gi;
+        const matches = regEx.exec(color);
+
+        if (matches) {
+            const r = parseInt(matches[1], 16);
+            const g = parseInt(matches[2], 16);
+            const b = parseInt(matches[3], 16);
+
+            return { r, g, b };
+        }
+    }
+
+    return penColors.black;
+}
+
+/**
+ * Lightens the given color by a certain intensity.
+ * @param color The color to lighten.
+ * @param intensity The intensity of the lightening. Must be between 0 and 100.
+ * @returns The lightened color.
+ */
+export function lightenColor(color: IColor, intensity: number): IColor {
+    if (intensity < 0 || intensity > 100) {
+        return color;
+    }
+
+    const brightenChannel = (channel: number) => {
+        const delta = 255 - channel;
+
+        return channel + delta / 100 * intensity;
+    }
+
+    return {
+        r: brightenChannel(color.r),
+        g: brightenChannel(color.g),
+        b: brightenChannel(color.b)
+    }
+}
+
+/**
+ * Darkens the given color by a certain intensity.
+ * @param color The color to darken.
+ * @param intensity The intensity of the darkening. Must be between 0 and 100.
+ * @returns The darkened color.
+ */
+ export function darkenColor(color: IColor, intensity: number): IColor {
+    if (intensity < 0 || intensity > 100) {
+        return color;
+    }
+
+    const darkenChannel = (channel: number) => {
+        const delta = 255 - channel;
+
+        return channel - channel / 100 * intensity;
+    }
+
+    return {
+        r: darkenChannel(color.r),
+        g: darkenChannel(color.g),
+        b: darkenChannel(color.b)
+    }
+}
+
+/**
+ * Pre-defined basic colors
+ */
+export const basicColors = {
+    black: fromCssColor("#000000"),
+    white: fromCssColor("#ffffff"),
+    gray: fromCssColor("#808080"),
+    silver: fromCssColor("#c0c0c0"),
+    red: fromCssColor("#ff0000"),
+    green: fromCssColor("#00ff00"),
+    blue: fromCssColor("#0000ff"),
+    yellow: fromCssColor("#ffff00"),
+    magenta: fromCssColor("#ea33f6"),
+    violet: fromCssColor("#8719cc"),
+    purple: fromCssColor("#76147d")
+};
+
+/**
+ * Pre-defined pen colors
+ */
+export const penColors = {
+    yellow: fromCssColor("#ffc114"),
+    orange: fromCssColor("#f6630d"),
+    pink: fromCssColor("#ff0066"),
+    red: fromCssColor("#e71224"),
+    indigo: fromCssColor("#5b2d90"),
+    purple: fromCssColor("#ab008b"),
+    plum: fromCssColor("#cc0066"),
+    darkBlue: fromCssColor("#004f8b"),
+    skyBlue: fromCssColor("#00a0d7"),
+    lightBlue: fromCssColor("#33ccff"),
+    green: fromCssColor("#008c3a"),
+    lightGreen: fromCssColor("#66cc00"),
+    black: fromCssColor("#000000"),
+    darkGray: fromCssColor("#333333"),
+    lightGray: fromCssColor("#849398"),
+    white: fromCssColor("#ffffff")
+};
+
+/**
+ * Pre-defined highlighter colors
+ */
+export const highlighterColors = {
```

Comment:
>highlighterColors [](http://example.com/codeflow?start=13&length=17)

same as above

Reply:
Fixed in https://github.com/microsoft/live-share-sdk/commit/18b13a631f0c2833a77b7d5181cb89b36206c512